README.md

```diff
@@ -15,6 +15,7 @@ Node.js versions [currently provided](https://hub.docker.com/r/bucharestgold/cen
 
 Version  | Tag
 -------- | -----
+`10.0.0` | (10.x-rc)
```

Comment:
Is this a typo? Should it be `cr`? Or is the typo below?

Reply:
Sorry, yes that is a typo. I've fixed that now. Thx